gphotocameraworker.cpp

```diff
@@ -488,10 +488,9 @@ void GPhotoCameraWorker::logOption(const char *name)
 void GPhotoCameraWorker::waitForOperationCompleted()
 {
     CameraEventType type;
-    void *data;
     int ret;
-
     do {
+        void *data=nullptr;
```

Comment:
Please, add the spaces around `=`, as we're trying to follow Qt code style guidelines. Otherwise, it's fine.

Reply:
Done